src/chart/components/pane/extent/y-extent-component.ts

```diff
@@ -46,8 +45,10 @@ export class YExtentComponent extends ChartBaseElement {
 		private hitTestController: PaneHitTestController,
 		public dataSeriesCanvasModel: CanvasModel,
 		public readonly scaleModel: ScaleModel,
-		// TODO when y-axis component will be refactored this shouldn't be undefined
-		public readonly yAxisComponent: ExtentYAxis | undefined,
+		createYAxisComponent: (
```

Comment:
If there'll be a factory as I proposed, pls, change the type here

Reply:
Could you tell, what type should be here?